package.yaml

```diff
@@ -44,23 +44,21 @@ dependencies:
 - cardano-sl-infra
 - cardano-sl-wallet
 
-library:
-  source-dirs: src
-  default-extensions:
-    OverloadedStrings
-    RecordWildCards
-    FlexibleContexts
-
 executables:
   dbgen:
     main:                Main.hs
-    source-dirs:         app
+    source-dirs:         src
```

Comment:
@ksaric Any reason why this was moved into `src`? Usually is cleaner to separate the executable with the library code.

Reply:
That's because of the bug - https://ghc.haskell.org/trac/ghc/ticket/13810
It doesn't build on Linux otherwise.

Reply:
Also, `RecordWildCards` seem enabled by default, can you delete the .cabal file and build again?

Reply:
Ah, I see, is when you try to build an artifact which depends from something compiled with the gold linker that it breaks down.